pkgs/tools/audio/google-music-scripts/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, python3 }:
+
+let click6 = python3.pkgs.callPackage ./click6 { };
```

Comment:
I'd do `python3.pkgs.callPackage` for `google-music-scripts` as well since it's only for python 3.

Reply:
Also why not use `overridePythonAttrs` for `click6`?